src/components/gallery/HdGalleryCarousel.vue

```diff
@@ -148,9 +148,9 @@ export default {
         this.updateCurrentIndex();
       });
     },
-    updateCurrentIndex(itemIndex) {
-      this.currentIndex = itemIndex;
-      this.$emit('input', itemIndex);
+    updateCurrentIndex() {
```

Comment:
```suggestion
    updateCurrentIndex(itemIndex) {
```

Reply:
Did not match with what I expected. However, after we've talked, all clear. :)